src/mutations/user.ts

```diff
@@ -14,6 +14,31 @@ export const changeUserPassword = gql`
   }
 `;
 
+export const registerAccount = gql`
+  mutation RegisterAccount($email: String!, $password: String!, $redirectUrl: String!) {
+    accountRegister(
+      input: { email: $email, password: $password, redirectUrl: $redirectUrl }
+    ) {
+      errors {
+        field
+        message
+      }
```

Comment:
Let's return error code if we can 🙃

```suggestion
      errors {
        code
        field
        message
      }
```